dds/DCPS/transport/rtps_udp/RtpsUdpTransport.cpp

```diff
@@ -648,12 +646,10 @@ namespace {
     if (result < 0) {
       const int err = errno;
       if (err != ENETUNREACH || !network_is_unreachable) {
-        ACE_TCHAR addr_buff[DCPS::AddrToStringSize] = {};
-        addr.addr_to_string(addr_buff, DCPS::AddrToStringSize);
         errno = err;
         const ACE_Log_Priority prio = shouldWarn(errno) ? LM_WARNING : LM_ERROR;
         ACE_ERROR((prio, "(%P|%t) RtpsUdpTransport.cpp send_single_i() - "
-                   "destination %s failed %p\n", addr_buff, ACE_TEXT("send")));
+                   "destination %C failed %p\n", DCPS::LogAddr(addr).c_str(), ACE_TEXT("send")));
```

Comment:
You're missing a few migrations from to_dds_string() (lines 841, 846, 875). I'd comment on them directly, but GitHub won't let me make a comment on a line that's not part of the diff.

Reply:
Good catch! Thanks.